ethereum/spec/src/main/java/tech/pegasys/teku/spec/executionengine/ExecutionEngineChannel.java

```diff
@@ -25,7 +25,6 @@
 import tech.pegasys.teku.spec.datastructures.execution.PowBlock;
 
 public interface ExecutionEngineChannel extends ChannelInterface {
-  String STUB_ENDPOINT_IDENTIFIER = "stub";
```

Comment:
Why did this move as part of this PR?

Reply:
I think I could do it other way, yeah

Reply:
No, it will go to `PowChainService` in this case as we have "some" execution engine endpoint and so we could use it for deposits info

Reply:
Ok with non-infrastructure module I could revert it back, thought it's still needed in a new module